instructions.md

```diff
@@ -0,0 +1,65 @@
+# instructions
+
+1: Use the virtual machine Docker to access the Grafana data source for the Cognite Data Platform. Download it here: https://www.docker.com/get-started
```

Comment:
`1.` to get a numbered list (you use dots for the others below, which is correct).